include/urbi/kernel/userver.hh

```diff
@@ -41,7 +41,7 @@
 # include <urbi/kernel/fwd.hh>
 # include <urbi/kernel/utypes.hh>
 # include <urbi/export.hh>
-# include <urbi/parser/location.hh>
+# include <parser/location.hh>
```

Comment:
This is probably the sign we should move these guys there at some point (the generated headers).

Maybe as a shorter workaround you could have left urbi/parser/location.hh as a simple `#include <parser/location.hh>`.  But that's only details.